taskfiles/launchpad_deps_linux.yml

```diff
@@ -82,35 +82,34 @@ tasks:
     cmds:
       - |
         cd $(mktemp -d)
-        wget https://github.com/derailed/k9s/releases/download/v{{.K9S_VERSION}}/k9s_Linux_x86_64.tar.gz
-        tar xfz k9s_Linux_x86_64.tar.gz
+        wget https://github.com/derailed/k9s/releases/download/v{{.K9S_VERSION}}/k9s_Linux_amd64.tar.gz
+        tar xfz k9s_Linux_amd64.tar.gz
```

Comment:
this change is because `k9s_Linux_x86_64.tar.gz` is no longer available in newer versions of K9S